app/api/attest/route.ts

```diff
@@ -34,8 +36,51 @@ export async function POST(req: NextRequest) {
     return NextResponse.json({ error: 'No wallet address found in session' }, { status: 400 })
   }
 
-  if (!isDiamondHands(walletAddress)) {
-    return NextResponse.json({ error: 'User is not diamond hands' }, { status: 400 })
+  const payload = await req.json()
+  let params;
+  switch(payload.schemaId) {
+
+    case DIAMOND_HANDS_SCHEMA_UID: {
+
+      if (!isDiamondHands(walletAddress)) {
+        return NextResponse.json({ error: 'User is not diamond hands' }, { status: 400 })
+      }
+
+      params = {
+        schema: DIAMOND_HANDS_SCHEMA_UID,
+        recipient: walletAddress,
+        expirationTime: 0n,
+        revocable: false,
+        refUID: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        data: DIAMOND_HANDS_ATTESTATION_DATA,
+        value: 0n,
+        deadline: 0n
+      };
+      break;
+
+    }
+    case TWITTER_SCHEMA_UID: {
+       // TODO Make sure they have a twitter ID in the JWT.
+       params = {
+        schema: TWITTER_SCHEMA_UID,
+        recipient: walletAddress,
+        expirationTime: 0n,
+        revocable: false,
+        refUID: '0x0000000000000000000000000000000000000000000000000000000000000000',
+        data: twitterEncoder.encodeData([{
+          name: 'twitterId',
+          type: 'uint256',
```

Comment:
Does twitter uses uint256 for user ids? I always assumed the user id was just the username

Reply:
So a user can change their username on twitter but the account ID is constant. There are simple API's to look up @username from the ID, I do need to double check uint256 is the right type to use for the ID though.

Reply:
Looks like we will need to make this change in the auth code to store account ID in jwt over handle.